data/data.rb

```diff
@@ -0,0 +1,172 @@
+require_relative '../book'
+require_relative '../person'
+require_relative '../teacher'
+require_relative '../student'
+require 'json'
+
+def save_book(title, author)
+  obj = {
+    title: title,
+    author: author
+  }
+
+  return unless File.exist?('./data/books.json')
+
+  file = File.open('./data/books.json')
+  # rubocop:disable Style/ZeroLengthPredicate
+  if file.size.zero?
+    book = [obj]
+  else
+    book = JSON.parse(File.read('./data/books.json'))
+    book << obj
+  end
+
+  file.close
+
+  addfile = File.open('./data/books.json', 'w')
+  addfile.write(JSON.generate(book))
+  addfile.close
+end
+
+def load_books
+  if File.exist?('./data/books.json')
+    file = File.open('./data/books.json')
+
+    if file.size.zero?
+      'No book records yet.'
+    else
+      books = JSON.parse(File.read('./data/books.json'))
+
+      books.each do |book|
+        book = Book.new(book['title'], book['author'])
+        @books << book
+      end
+    end
+
+    file.close
+
+  else
+    puts 'Please add books.'
+  end
+  puts 'Books:'
+  @books.each { |b| puts "Book Title: #{b.title}, Book Author: #{b.author}" } unless @books.empty?
+end
+
+def load_person
+  if File.exist?('./data/person.json')
+    file = File.open('./data/person.json')
+    if File.empty?('./data/person.json')
+      puts 'No person records yet.'
+    else
+      people = JSON.parse(File.read('./data/person.json'))
+      people.each do |person|
+        if person['option'] == 'Student'
+          student = Student.new(person['age'], person['name'])
+          @people << student
+        else
+          teacher = Teacher.new(person['specialization'], person['age'], person['name'])
+          @people << teacher
+        end
+      end
+    end
+    file.close
+  else
+    puts 'Please add a person.'
+  end
+end
```

Comment:
- [ ] Kindly make sure to load all people once the app starts running. You're loading other items correctly but people are not being displayed. Kindly fix that to meet that project requirement.

   - **YOUR APP**
   
    ![image](https://user-images.githubusercontent.com/87197412/210328438-c6491dea-9ca2-44f0-9983-983b42a32190.png)